cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -18,24 +18,24 @@ personas:
 
 clusterPackages:
```

Comment:
Note: we still don't have `alibaba` here, but since it is not on `master` currently I opted not to add it here.